src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -863,3 +877,41 @@ function getQuestionIdSuffix(questionNum) {
     return idSuffix;
 }
 
+function toggleCustomFeedbackPathsDisplay(toggleLink) {
+    var $toggleLink = $(toggleLink);
+    var $customFeedbackPathsDisplay = $toggleLink.closest('.custom-feedback-paths-row')
+                                                 .find('.custom-feedback-paths-display');
+    if ($customFeedbackPathsDisplay.is(':visible')) {
+        $customFeedbackPathsDisplay.hide();
+        $toggleLink.text('Show details and further customizations');
+    } else {
+        $customFeedbackPathsDisplay.show();
+        $toggleLink.text('Hide details and further customizations');
+    }
+}
+
+function regenerateCustomFeedbackPathsSpreadsheet(questionNum) {
+    var data = customFeedbackPathsDataForEachQuestionBeforeEdit[questionNum];
+    var $container = $('#questionTable' + questionNum + ' .custom-feedback-paths-spreadsheet');
+    $container.handsontable({
+        data: data,
+        minRows: 15,
+        minCols: 2,
+        minSpareRows: 1,
+        rowHeaders: true,
+        colHeaders: ['Feedback giver', 'Feedback recipient'],
+        columns: [{ readOnly: true }, { readOnly: true }],
+        manualColumnResize: true,
+        manualRowResize: true,
+        stretchH: 'all'
+    });
+}
+
+function removeUnselectedCustomOptions() {
```

Comment:
Can we remove these in the `InstructorFeedbackEditPageData` when generating the options tag? Somewhere near `getParticipantOptions()`